BankSDK/GiniBankSDK/Documentation/source/Migration to version 3.x.x.md

```diff
@@ -285,11 +285,64 @@ Showing errors during usage of the SDK was changed from snackbar to a whole new
 
 The new error screen gives options to retake photos or enter details manually and displays errors with more detailed description.
 
-You can find more details [here](https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/3.0.0-beta06/customization-guide.html#error-screen).
+You can find more details [here](https://developer.gini.net/gini-mobile-ios/GiniBankSDK/3.0.0-beta06/customization-guide.html#error-screen).
 
 #### Option to enter details manually
 
 You can show your own UI for data input if an error occured and the user clicks the "Enter manually" button on the error screen.
 For this you must to implement `GiniCaptureResultsDelegate.giniCaptureDidEnterManually() `.
 
-You can find more details [here](https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/3.0.0-beta06/features.html#error-screen-customization) and [here]((https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/3.0.0-beta06/customization-guide.html#error-screen).
+You can find more details [here](https://developer.gini.net/gini-mobile-ios/GiniBankSDK/3.0.0-beta06/features.html#error-screen-customization) and [here]((https://developer.gini.net/gini-mobile-ios/GiniBankSDK/3.0.0-beta06/customization-guide.html#error-screen).
+
+# Migrate Return Assistant
+
+## Breaking Changes
+
+Info box was removed.
```

Comment:
[Suggestion] I think we should mention the breaking changes for each screen separately. So this info box removal breaking change should go to the overview screen's breaking changes section.

Reply:
@a-szotyori yes, good point. Thanks